app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayGridViewHolder.kt

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.tabstray
```

Comment:
nit: we can move these to the `org.mozilla.fenix.tabstray.browser` package since it's for the "browser" tabs.

That should help when we have the `syncedtabs` package for the Synced Tabs viewholders, etc.

Reply:
Sure, I'll move this once this is merged.